src/commands/login.io.ts

```diff
@@ -1,6 +1,28 @@
 import prompts from "prompts";
 import { C } from "../utils/screens.js";
 
+export const promptForOrigin = async () => {
+  const ownOrigin = await prompts({
+    type: 'confirm',
+    name: 'value',
+    message: 'Would you like the use custom origin?',
+    initial: false
+  });
+
+  return Boolean(ownOrigin.value);
+};
+
+export const promptForOriginValue = async () => {
+  const originInput = await prompts({
+    type: 'text',
+    name: 'value',
+    message: 'Enter Monokle remote web app instance URL',
```

Comment:
Would plain hostname also work? e.g.`monokle.example.com`. This makes it easier for the end users by not having to type the URL scheme. (We can still normalise it to an origin for internal usage afterwards)

`Enter Monokle Enterprise hostname:`

Reply:
Yes, you can use hostname without protocol like `app.staging.monokle.com`. It's normalized by synchronizer.